build.sbt

```diff
@@ -200,6 +202,79 @@ lazy val sdkSpringTestKit = project
       "java.lang"))
   .settings(Dependencies.sdkSpringTestKit)
 
+lazy val sdkSpringBootStarter = project
+  .in(file("sdk/spring-boot-starter"))
+  .dependsOn(sdkSpring)
+  .enablePlugins(BuildInfoPlugin, PublishSonatype)
+  .settings(common)
+  .settings(
+    name := "kalix-spring-boot-starter",
```

Comment:
I'm a bit skeptical if we should tie this to springboot rather than keeping it as spring... if there is any remote chance that we might need to include anything spring-related and not springboot specific in here, I would probably just go with spring. E.g. I'm not sure where the parsing of the spring mvc annotations will end up, but those are still spring concepts, not spring boot ones.

Reply:
A starter is a Spring Boot concept and won't work with just Spring. 

The parsing of annotations is indeed generic and is not be in `kalix-spring-boot-starter`.

Reply:
So, what your thinking is that we'll eventually get ride of the spring-sdk module, and we just have the `kalix-spring-boot-starter` that will depend directly on the `java-sdk`.. is this correct?

Reply:
More or less. I still think that we will need to have something in between, but maybe something called `kalix-spring`. This module will contain the Spring annotations scanner and some other support classes. 

Starters are more like more similar to configuration package. It provides transitive dependencies and a `@AutoConfiguration` class to be loaded. 

For the moment, `KalixConfiguration` didn't move to the starter because we use it in our tests, but we can re-think it later as well.

Reply:
Ok, cool. One thing I'm thinking about is package imports on the end-user kalix apps..

Do we foresee having something like this?
```java
import kalix.javasdk.Something;
import kalix.spring.AnotherOne;
import kalix.springboot.YetAnotherOne;
```

If this is a possibility, maybe having the below would be better:
```java
import kalix.javasdk.Something;
import kalix.spring.AnotherOne;
import kalix.spring.boot.YetAnotherOne;
```

Note `springboot` vs `spring.boot`.. This would also be consistent with the springboot packages themselves that do `springframework.boot`.

Anyway, this is nitpicking but I know you also care about this. :D

Reply:
I have been thinking about that as well. `springboot` vs. `spring.boot`. 
I looked at other starters and there is also a mix. 

Maybe indeed better for us to have `kalix.spring` and `kalix.spring.boot`.

Reply:
renamed to `kalix.spring.boot`